Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/outbound/StandardOutboundPatientDiscoveryTest.java

```diff
@@ -56,4 +90,129 @@ public void hasOutboundProcessingEvent() throws Exception {
+        NhinTargetSystemType target = MessageGeneratorUtils.getInstance().convertFirstToNhinTargetSystemType(targets);
+        AssertionType assertion = createAssertion();
+        request.setAssertion(assertion);
+        final OutboundPatientDiscoveryOrchestratable mockOrchestratable = mock(OutboundPatientDiscoveryOrchestratable.class);
+        //outOrchestratable.setResponse(new PRPAIN201306UV02());
+        final PatientDiscoveryAuditLogger auditLogger = getAuditLogger(true);
+        PatientDiscoveryPolicyChecker pdPolicyChecker = mock(PatientDiscoveryPolicyChecker.class);
+
+        PolicyEngineProxyObjectFactory policyFactory = mock(PolicyEngineProxyObjectFactory.class);
+        PolicyEngineProxy policyEngine = mock(PolicyEngineProxy.class);
+
+        //when(delegate.process(any(OutboundPatientDiscoveryOrchestratable.class))).thenReturn(outOrchestratable);
+        when(pdPolicyChecker.checkOutgoingPolicy(any(RespondingGatewayPRPAIN201305UV02RequestType.class))).thenReturn(Boolean.TRUE);
+        when(policyFactory.create()).thenReturn(policyEngine);
+
+        StandardOutboundPatientDiscovery standardPD = new StandardOutboundPatientDiscovery() {
+            @Override
+            protected PatientDiscoveryAuditLogger getNewPatientDiscoveryAuditLogger() {
+                return auditLogger;
+            }
+
+            @Override
+            protected List<UrlInfo> getEndpoints(NhinTargetCommunitiesType targetCommunities) {
+                return getEndPoints(MessageGeneratorUtils.getInstance().convertFirstToNhinTargetSystemType(
+                    targetCommunities));
+            }
+
+            @Override
+            protected boolean checkPolicy(RespondingGatewayPRPAIN201305UV02RequestType request) {
+                return true;
+            }
+
+            @Override
+            protected OutboundPatientDiscoveryOrchestratable createOrchestratable(PRPAIN201305UV02 message,
+                AssertionType assertion, NhinTargetSystemType target, NhincConstants.GATEWAY_API_LEVEL gatewayLevel) {
+                return mockOrchestratable;
+            }
+        };
+        when(mockOrchestratable.getAssertion()).thenReturn(assertion);
+        when(mockOrchestratable.getRequest()).thenReturn(request.getPRPAIN201305UV02());
+        when(mockOrchestratable.getTarget()).thenReturn(target);
+        RespondingGatewayPRPAIN201306UV02ResponseType actualMessage = standardPD.respondingGatewayPRPAIN201305UV02(
+            request, assertion);
+
+        verify(mockEJBLogger).auditRequestMessage(eq(request.getPRPAIN201305UV02()), any(AssertionType.class), any(
+            NhinTargetSystemType.class), eq(NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION),
+            eq(NhincConstants.AUDIT_LOG_NHIN_INTERFACE), eq(Boolean.TRUE), isNull(Properties.class),
+            eq(NhincConstants.PATIENT_DISCOVERY_SERVICE_NAME), any(PatientDiscoveryAuditTransforms.class));
+    }
+
+    /*@Test
```

Comment:
Lots of commented code in this class (above and below).  Are these going to be uncommented later?